site/content/en/blog/releases/1.10.0-rc.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "1.10.0 - Release Candidate"
+linkTitle: "1.10.0-rc"
+date: "2020-11-03"
+---
+
+This is the 1.10.0-rc release of Agones.
+
+This release fixes critical performance bugs in the gRPC Allocation endpoint, as well as fixing a memory leak
+inherited from client-go. Therefore, we recommend upgrading your install of Agones as soon as possible.
```

Comment:
Are you going to repeat this note in the stable release next week? I'm not sure we should encourage upgrading to a release candidate as soon as possible....

Reply:
Yeah, I usually repeat the same note between RC and stable.